cmd/agent/app/snmp.go

```diff
@@ -314,3 +320,19 @@ func printValue(pdu gosnmp.SnmpPDU) error {
 
 	return nil
 }
+
+var strippableSpecialChars = map[byte]bool{'\r': true, '\n': true, '\t': true}
+
+// IsStringPrintable returns true if the provided byte array is only composed of printable characeters
+func IsStringPrintable(bytesValue []byte) bool {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Nice to have, this function could be unit tested.